services/application-mapper/index.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const ApplicationQuestion = require('./application-question');
+const addressDetailsColumns = require('../../constants/address-details-columns');
+const mapTimestampToTariffFormat = require('./date-part-mapper');
+
+// The initial oracle object
+const oracleJsonObject = {
+    tables: [
+        {
+            APPLICATION_FORM: {
+                prefix: 'U',
+                section_ref: 'TEMP'
+            }
+        },
+        {
+            // TO-DO This will be dynamically set in the future
+            ADDRESS_DETAILS: {
+                address_type: 'ICA'
+            }
+        }
+    ]
+};
+
+async function mapApplicationDataToOracleObject(data) {
+    const applicationFormJson = Object.values(oracleJsonObject)[0][0].APPLICATION_FORM;
+    const addressDetailsJson = Object.values(oracleJsonObject)[0][1].ADDRESS_DETAILS;
```

Comment:
This will need to change soon in order to support multiple insertions to the address table with different address types in a single run of the function so I will take this into account then.